packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -310,12 +397,19 @@ export default mixins(genericHelpers, nodeHelpers, restApi, showMessage).extend(
 			this.$emit('credentialSelected', updateInformation);
 		},
 
-		onCredentialSelected(credentialType: string, credentialId: string | null | undefined) {
+		onCredentialSelected(
+			credentialType: string,
+			credentialId: string | null | undefined,
+			requiredCredentials = false,
```

Comment:
Naming is a bit confusing, I'd expect an array of creds instead of a boolean.

Reply:
Agree. Changing to `areCredentialsRequired`